src/Config.cpp

```diff
@@ -0,0 +1,141 @@
+#include <cstddef>
+
+#include "Config.hpp"
+#include "ErrorOr.hpp"
+#include "json-parser/JSONParser.hpp"
+#include "json-parser/JSONValue.hpp"
+
+namespace Config {
+  namespace {
+    auto parseRecipe(
+      FileConfig &conf,
+      const std::string &key,
+      const JSON::JSONValue &value) -> MaybeError
+    {
+      conf.recipesIds->add(key);
+      size_t recipe_id = TRY(conf.recipesIds->lookup(key));
+
+      std::vector<size_t> recipeContent;
+
+      JSON::JSONObject recipe_data = TRY(value.get<JSON::JSONObject>());
```

Comment:
```suggestion
      auto recipe_data = TRY(value.get<JSON::JSONObject>());
```
and every other below